crates/bevy_ui/src/layout/mod.rs

```diff
@@ -393,6 +414,21 @@ fn round_layout_coords(value: Vec2) -> Vec2 {
     }
 }
 
+/// Resolve the border thickness on a UI node's edge
+/// Both vertical and horizontal percentage border values are calculated based on the width of the parent node
+/// <https://developer.mozilla.org/en-US/docs/Web/CSS/border-width>
+fn resolve_border_thickness(value: Val, parent_width: f32, viewport_size: Vec2) -> f32 {
```

Comment:
I think renaming `value` to `border_value` or something would make this a bit more clear.

Reply:
yeah, `edge_value` I think would make sense